tests/ert3/config/test_stages_config.py

```diff
@@ -48,46 +63,21 @@ def test_entry_point_not_valid(config, expected_error):
         ert3.config.load_stages_config(config)
 
 
-def test_step_valid():
-    config = ert3.config.load_stages_config(
-        [
-            {
-                "name": "some_name",
-                "type": "unix",
-                "input": [{"record": "some_record", "location": "some_location"}],
-                "output": [{"record": "some_record", "location": "some_location"}],
-                "transportable_commands": [{"name": "poly", "location": "poly.py"}],
-                "script": ["poly --help"],
-            }
-        ]
-    )
-    assert config[0].name == "some_name"
+def test_step_valid(base_unix_stage_config):
+    config = ert3.config.load_stages_config(base_unix_stage_config)
+    assert config[0].name == "unix_stage"
     assert config[0].script[0] == "poly --help"
 
 
-def test_single_function_step_valid():
-    config = ert3.config.load_stages_config(
-        [
-            {
-                "name": "minimal_function_stage",
-                "type": "function",
-                "input": [{"record": "some_record", "location": "some_location"}],
-                "output": [{"record": "some_record", "location": "some_location"}],
-                "function": "builtins:sum",
-            }
-        ]
-    )
-    assert config[0].name == "minimal_function_stage"
+def test_single_function_step_valid(base_function_stage_config):
+    config = ert3.config.load_stages_config(base_function_stage_config)
+    assert config[0].name == "function_stage"
     assert isinstance(config[0].function, Callable)
     assert config[0].function.__name__ == "sum"
 
 
-def test_step_multi_cmd(tmpdir):
-    tmpdir.chdir()
-    shutil.copy2(_POLY_EXEC, "poly.py")
-    shutil.copy2(_POLY_EXEC, "poly2")
-
-    config = _example_config()
+def test_step_multi_cmd(base_unix_stage_config):
```

Comment:
Does this test need some assert, to make sure both commands are there after loading?

Reply:
It does not need it but I guess it could be nice to have.